### DISCUSSION TITLE
Which commit does version 2.58 come from?

### DISCUSSION THREAD

**Discussion by Cyberoliv** [shridhar-tl/jira-assistant#364] [Q&A] [answered]
Hi 🙂 
We are studying this application in order to integrate it into our Jira process.
Security asks us to take a look...
In order to continue the study I need to know the commit of the last published version.
Please note that we work with Google Chrome.
The latest tag I find in Github is 2.57 while the version in the Web Store is 2.58

Could you please tell me which commit was used to tag version 2.58?

Thank you in advance for your answers

Kind regards

**Comment by shridhar-tl** [comment, marked as answer]
Hi @Cyberoliv, I just realized I haven't tagged v2.58. However looking at the commit details and deployment date, It appears to be like the following commit should be the latest commit available in v2.58.

https://github.com/shridhar-tl/jira-assistant/tree/83e5a0b9e419bfbbe9e947fd501fed5e6a378d2c.

**Comment by Cyberoliv** [comment]
Thank you for your feedback, well noted.

Kind regards,

Olivier
